lib/cldr/export/data/numbers.rb

```diff
@@ -41,8 +41,13 @@ def currency
         end
 
         def symbols
-          select("numbers/symbols/*").each_with_object({}) do |node, result|
-            result[name(node).to_sym] = node.content
+          select("numbers/symbols").each_with_object({}) do |node, result|
+            next if node.attribute("numberSystem").nil? || node.attribute("numberSystem").value != "latn"
```

Comment:
```suggestion
          select("numbers/symbols[@numberSystem=\"latn\"]").each_with_object({}) do |node, result|
```

Use XPath to select the specific one. At which point, I think this PR becomes the single line change?

Reply:
Thanks, I've updated it to "numbers/symbols[@numberSystem=\"latn\"]/*" to make it single line change.